recipes/wannierberri/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "wannierberri" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stepan-tsirkin/wannier-berri/archive/master.tar.gz
+  sha256: bec13eda36db456a47f01ce56f15bed17ad93dc8f4baceb6c031146b8238644e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
+    - scipy
+    - lazy_property
+    - pyfftw
+    - colorama
+    - termcolor
+    - pyfiglet
```

Comment:
Are all of these needed at package build time?

Reply:
The issue is they import the package itself during setup https://github.com/stepan-tsirkin/wannier-berri/blob/master/setup.py#L9 so I basically need all dependencies already during the installation.